python/llm/src/ipex_llm/transformers/npu_models/baichuan_mp.py

```diff
@@ -729,13 +724,14 @@ def forward(
             for i in range(len(self.decoder_processes)):
                 self.input_queues[i].put(past_key_value)
 
-        control = torch.tensor(0, dtype=torch.int)
-        dist.broadcast(control, src=0)
+        t0 = time.perf_counter()
```

Comment:
Maybe we could also remove this ? Other LGTM

Reply:
done